hikari/impl/interaction_server.py

```diff
@@ -176,7 +189,11 @@ def __init__(
         self._loads = loads
         self._rest_client = rest_client
         self._server: typing.Optional[aiohttp.web_runner.AppRunner] = None
-        self._verify = ed25519.build_ed25519_verifier(public_key) if public_key is not None else None
+
+        # if not isinstance(public_key, bytes) or len(public_key) != 32:
+        #     raise ValueError("public_key must be a 32-byte sequence.")
```

Comment:
Any reason for this being commented out?

Reply:
Because i haven't poked around how pynacl handles those cases itself yet